Robot/src/main/java/com/swrobotics/robot/commands/AimAtLimelight2.java

```diff
@@ -0,0 +1,27 @@
+package com.swrobotics.robot.commands;
+
+import com.swrobotics.lib.swerve.commands.TurnToAngleCommand;
+import com.swrobotics.mathlib.Angle;
+import com.swrobotics.robot.RobotContainer;
+import com.swrobotics.robot.subsystems.vision.Limelight;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+import java.util.function.Supplier;
+
+public class AimAtLimelight2 extends TurnToAngleCommand {
+    private final Limelight limelight;
+
+
+    public AimAtLimelight2(RobotContainer robot, Supplier<Angle> angle) {
+        super(robot, angle, true);
```

Comment:
Can probably just pass null for angle supplier since you are setting it manually in execute()